src/OneExplorer/ConfigObject.ts

```diff
@@ -112,12 +121,31 @@ export class ConfigObj {
     return found ? true : false;
   }
 
+  /**
+   * @brief getter for config setting
+   */
+  public get configSetting(): ConfigSetting {
+    let configSetting: ConfigSetting;
+    switch (this.configType) {
+      case CfgType.one:
+      default:
+        configSetting = new OneConfigSetting();
+    }
+    configSetting.init();
+
+    return configSetting;
+  }
+
   private constructor(uri: vscode.Uri, rawObj: Cfg) {
     this.uri = uri;
     this.rawObj = rawObj;
+    this.configType = CfgType.one;
+
+    // separate to init()
```

Comment:
```suggestion
    // TODO: separate to init()
```

Reply:
Could you just leave this `TODO` together?

Reply:
Thank you! I will add `TODO` comment.